tests/functional/processes/frbr/test_frbr_process.py

```diff
@@ -16,3 +15,15 @@ def test_frbr_process(db_manager, unfrbrized_record_uuid):
     frbrized_record = db_manager.session.query(Record).filter(Record.uuid == unfrbrized_record_uuid).first()
     
     assert frbrized_record.frbr_status == 'complete'
+
+    db_manager.session.commit()
+    db_manager.session.expire_all()
+
+    test_records = db_manager.session.query(Record).filter(Record.source == 'oclcClassify').all()
```

Comment:
Where is this `test_records` variable utilized in the code after its creation? I'm wondering the purpose of this variable.